apis/datadoghq/v2alpha1/condition.go

```diff
@@ -3,9 +3,29 @@ package v2alpha1
 import (
 	"fmt"
 
+	apicommon "github.com/DataDog/datadog-operator/apis/datadoghq/common"
+
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+// DatadogAgentState type representing the deployment state of the different Agent components.
+type DatadogAgentState string
+
+const (
+	// DatadogAgentStateProgressing the deployment is running properly.
```

Comment:
```suggestion
	// DatadogAgentStateProgressing the deployment is progressing.
```
or something similar?